### PR TITLE
README Edit - Lori

### PR DESCRIPTION
**Pull request by Lbeckap**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,3 +1,5 @@
 This app lets you create full page slideshows by combining images, text markup, and animated transitions.
 
 Todo: after the slideshow has been created, the app should be able to produce a monolithic HTML file that replicates the slideshow without the cpontrol panel overlay
+
+README TEST - Lori
```
Comment (Lbeckap) [inline README:5]: This was a simple text edit to the readme file.

File: indexScratch.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+<head>
+    <title>Parallax Slider with Form</title>
+   <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <div id="slider" class="swipe">
+        <div class="swipe-wrap">
+
+            <!--START COMMENTED OUT TRAINING SLIDES-->
```
Comment (Lbeckap) [inline indexScratch.html:12]: Commented out text will be removed after reviewed with instructor.

File: scratch.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+// new Swipe(slider, {
+//     speed: 600,
+//     auto: 3000,
+//     continuous: true
+//   })
+
```
Comment (Lbeckap) [inline scratch.js:6]: This file should be a copy of the original script.js. New functions were only added to training.js at this time.

File: splash.css [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    min-width: 300px;
+    height: 35vh; 
+    min-height: 300px;
+    background-color: #d5c3a9eb;
+    border-radius: 15px;
+    padding: 20px;
+    margin: auto;
+    top: 50%;
+    transform: translate(0, +50%);
+    /* border-style: solid;
```
Comment (Lbeckap) [inline splash.css:57]: Will ask for more opinion on if border looks nice or not.

File: training.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
+// new Swipe(slider, {
+//     speed: 600,
+//     auto: 3000,
+//     continuous: true
+//   })
+
+
+trainingDiv1();
```
Comment (Lbeckap) [inline training.js:8]: New functions to show hard coded training slides.

**Review by Lbeckap — COMMENTED** [review]
All unused code has been commented out and will be removed after instructor review.

File: indexScratch.html (hunk 2 above)
Comment (rharward) [reply indexScratch.html:12]: Great, looks fine for a test of the Pull Request functionality.

**Comment by basicmammal23** [comment]
I appreciate all the creative additions and that you've added a comment per addition to the code. This makes it easy to read and follow. Commenting out unused code is also a smart idea!

**Pull request merged**